io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -71,17 +76,18 @@ private:
    TBufferMerger(const TBufferMerger &);
 
    /** TBufferMerger has no copy operator */
-   TBufferMerger& operator=(const TBufferMerger&);
+   TBufferMerger &operator=(const TBufferMerger &);
 
    void Push(TBufferFile *buffer);
-   void Listen();
+   void WriteOutputFile();
 
-   std::mutex fFilesMutex;                                       //< Mutex used to lock fAttachedFiles
+   const std::string fName;
+   const std::string fOption;
+   const Int_t fCompress;
```

Comment:
> Is there a particular reason to mark the members only const?
It insures that the invariant of the class (not-allowed to change output file after the class) are guaranteed.  It also allows/indicates-that-you-are-allowed to use those from multiple thread without risk of race condition.